templates/AureliaSpa/ClientApp/app/components/fetchdata/fetchdata.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/whatwg-fetch.d.ts" />
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/url.d.ts" />
```

Comment:
It's unfortunate that these are necessary. Has the underlying issue in Aurelia been reported? Is there any chance of getting a fix published?